tests/mmkubernetes-basic-vg.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# added 2018-04-06 by richm, released under ASL 2.0
+#export RSYSLOG_DEBUG="debug"
+. $srcdir/diag.sh init
+
+python ./mmkubernetes_test_server.py 18443 > mmk8s_srv.log 2>&1 &
+BGPROCESS=$!
```

Comment:
do we need a check here that the startup really was completed? I ask because we have some other spots inside the testbench where this causes startup races, especially on slow machines. side-note: we may want to consider adding some generic plumbing to the testbench to do this - so maybe we should just file an enhancement issue for now (if this is a problem at all).

Reply:
I was able to use the plumbing in diag.sh - how's this?